force-app/main/default/lwc/datatable/datatable.js

```diff
@@ -29,7 +31,8 @@ export default class Datatable extends LightningElement {
   _recordsPerBatch;
   _recordCount;
   _tableRequest = '';
-  _columnsToHide = {};
```

Comment:
@shliachtx , why is this being removed? The idea of this property is so that certain columns can be included in the query but not displayed in the list view. For example, Id and related record Id - when one wants to show the name with a hyperlink we need to query both the id and the name but we don't want to show the id field. (There are other examples but this is the simplest)

Reply:
Never mind, seems this is not needed because we have a "visible" property on each field.